File: isegoria/src/main/java/com/eulersbridge/isegoria/models/Candidate.java
package com.eulersbridge.isegoria.models;

import com.squareup.moshi.Json;

public class Candidate {

    @Json(name = "candidateId")
    public long id;
    public long ticketId;
    public long positionId;
    public long userId;

    public String familyName;
    public String givenName;
    public String policyStatement;
    public String information;

    public String getName() {
        return String.format("%s %s", givenName, familyName);
    }

}

File: isegoria/src/main/kotlin/com/eulersbridge/isegoria/network/api/model/Candidate.kt
package com.eulersbridge.isegoria.network.api.model

import android.annotation.SuppressLint
import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@SuppressLint("ParcelCreator")
@Parcelize
data class Candidate (
    @field:Json(name = "candidateId")
    val id: Long = 0,

    val ticketId: Long?,
    val positionId: Long = 0,
    val userId: Long = 0,
    val familyName: String?,
    val givenName: String?,
    val policyStatement: String?,
    val information: String?,
    val userProfile: User

) : Parcelable {
    val name: String
        get() = "$givenName $familyName"
}